### PR TITLE
[minor] Add support for OCP 4.13, OCP 4.14 and remove support for OCP 4.10

### PR DESCRIPTION
**Pull request by istrate**
### Description

- Work part of adding support OCP 4.13.x and OCP 4.14.x
- Making OCP 4.14 version as default
- Removing support for OCP 4.10

**Note to reviewers: Not to be merged until after final decision on when to GA OCP 4.13 and 4.14 support in MAS**

### CODE REVIEW

File: docs/catalogs/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ As the MAS CLI is updated we maintain a rolling window of approximately four mon
...
 
-All catalogs are available **indefinitely**, but they have a useful lifespan limited by the support statements of the packages available in the catalog and the OCP release the catalog is certified on.  We **never** remove catalog images from the IBM Container Registry (ICR), even the first Maximo Operator Catalog ever published - [v8-220717-amd64](v8-220717-amd64.md) - is still available today, however it's usefulness is questionable due to the end of life of all compatible OCP releases.
+All catalogs are available **indefinitely**, but they have a useful lifespan limited by the support statements of the packages available in the catalog and the OCP release the catalog is certified on.  We **never** remove catalog images from the IBM Container Registry (ICR), even the first Maximo Operator Catalog ever published - [v8-231228-amd64](v8-231228-amd64.md) - is still available today, however it's usefulness is questionable due to the end of life of all compatible OCP releases.
```
Comment (sanjayprab) [inline docs/catalogs/index.md]: the statement says, first Maximo operator catalog ever published, 231228 is not that.. could we revert this??

File: image/cli/mascli/functions/provision_aws [hunk 2]
```diff
@@ -137,12 +137,12 @@ function provision_aws_interactive() {
   # prompt_ocp_version                  # OCP_VERSION
   echo
   echo "OCP Version:"
-  echo "  1. 4.10 (MAS 8.9-8.10)"
+  echo "  1. 4.14 (MAS 8.09-8.11)"
```
Comment (sanjayprab) [inline image/cli/mascli/functions/provision_aws]: should be 8.10 - 8.11

File: image/cli/mascli/functions/provision_roks [hunk 3]
```diff
@@ -123,12 +123,12 @@ function provision_roks_interactive() {
 
   echo
   echo "OCP Version:"
-  echo "  1. 4.10 (MAS 8.9-8.10)"
+  echo "  1. 4.14 (MAS 8.9-8.10)"
```
Comment (sanjayprab) [inline image/cli/mascli/functions/provision_roks]: 4.14 supports from 8.10 - 8.11

**Review by whitfiea — CHANGES_REQUESTED** [review]
Can we add to the PR comment and title that this is also removing 4.10 support

File: image/cli/mascli/functions/provision_roks (hunk 3 above)
Comment (istrate) [reply image/cli/mascli/functions/provision_roks]: done!

File: image/cli/mascli/functions/provision_aws (hunk 2 above)
Comment (istrate) [reply image/cli/mascli/functions/provision_aws]: done!

File: docs/catalogs/index.md (hunk 1 above)
Comment (istrate) [reply docs/catalogs/index.md]: done!

File: image/cli/mascli/functions/provision_fyre [hunk 4]
```diff
@@ -129,16 +129,16 @@ function provision_fyre_interactive() {
...
   case $OCP_VERSION_SELECTION in
-    1|4.10)
-      export OCP_VERSION=4.10
+    1|4.12)
+      export OCP_VERSION=4.12
       ;;
-    2|4.12)
-      export OCP_VERSION=4.12.22
+    2|4.14)
+      export OCP_VERSION=4.14.5
```
Comment (sanjayprab) [inline image/cli/mascli/functions/provision_fyre]: shall we keep this as 4.14.12

**Review by sanjayprab — APPROVED** [review]

**Review by whitfiea — APPROVED** [review]

**Pull request merged**